application/backend/templates/emails/activate-account.html.twig

```diff
@@ -0,0 +1,8 @@
+{% extends 'emails/base_email.html.twig' %}
+
+{% block content %}
+    <p>{{ 'email.activate-account.content.hello'|trans }} {{ user.fullName }},<br>
```

Comment:
Perhaps put the `hello` word in a general translation rather than in this mail-specific domain. 
We'll end up with 10 hello in translation otherwise (and maybe change also the other mail to use this translation)